src/components/MapView/Legends/index.tsx

```diff
@@ -43,6 +62,14 @@ function Legends({ classes, layers }: LegendsProps) {
           ? formatWMSLegendUrl(layer.baseUrl, layer.serverLayerName)
           : undefined;
 
+      // Population exposure analysis will run only for polygon layers.
+      const exposure =
```

Comment:
Consider making a function `getExposureFromLayer` if you think there's future use for this. Otherwise if its one-use this is fine :+1:

Reply:
Sure. Included in 29c0742f2138e556f90cdf9afea1a9e8928b5079